src/backend/expungeservice/models/charge.py

```diff
@@ -57,9 +57,9 @@ def set_time_ineligible(self, reason, date_of_eligibility):
             date_will_be_eligible = date_of_eligibility
         else:
             date_will_be_eligible = None
-        time_eligibility = TimeEligibility(status=False, reason=reason, date_will_be_eligible=date_will_be_eligible)
+        time_eligibility = TimeEligibility(status = EligibilityStatus.INELIGIBLE, reason = reason, date_will_be_eligible = date_will_be_eligible)
```

Comment:
There should not be spaces around the equal sign per the pep 8 style guide.

> Don't use spaces around the = sign when used to indicate a keyword argument,

Reply:
We may want to consider adding a post-commit hook for [`black`](https://black.readthedocs.io/en/stable/).